rllib/models/torch/torch_distributions.py

```diff
@@ -220,7 +220,12 @@ def sample(
         if sample_shape is None:
             sample_shape = torch.Size()
         loc_shape = self.loc.shape
-        return torch.ones(sample_shape + loc_shape, device=self.loc.device) * self.loc
+        return (
```

Comment:
was left out from the action_dist PR :)

Reply:
is lint broken in master?
you should probably send a separate PR so we can quickly merge this if so.

Reply:
https://github.com/ray-project/ray/pull/29648

Reply:
I don't understand why lint didn't pass here. The file def. has a `__main__`